src/main.zig

```diff
@@ -7,10 +7,8 @@ const stb = @cImport({
     @cInclude("stb_image_resize2.h");
 });
 
-// VARS
-const ASCII_CHARS = " .:-=+*%#@"; // low to high brightness
-const CHAR_SIZE = 8;
-const THRESHOLD = 50;
+const default_characters = " .:-=+*%@#";
+const full_characters = " .-:=+iltIcsv1x%7aejorzfnuCJT3*69LYpqy25SbdgFGOVXkPhmw48AQDEHKUZR@B#NW0M";
```

Comment:
I think the full characters don't look that good. The dots above the j overlap other pixels. Our default ascii characters should ideally be perfect.

Let's hold off on the full character list.

We should instead explore a Unicode character list instead.

Reply:
![CleanShot 2024-09-02 at 23 56 13@2x](https://github.com/user-attachments/assets/633a2d56-7c4b-4dc4-994f-ee3552de8e45)